api/v1/helpers/verbs/utils.js

```diff
@@ -617,6 +618,8 @@ function findByKey(props, params, extraAttributes) {
   const scopedModel = getScopedModel(props, attrArr);
   if (looksLikeId(key)) {
     return findByIdThenName(scopedModel, key, opts);
+  } else if ((typeof key === 'number') && (key % 1 === 0)) {
```

Comment:
can you add a comment in the code since it isn't immediately obvious why you're adding this here?